gym_electric_motor/physical_systems/electric_motors/externally_excited_synchronous_motor.py

```diff
@@ -167,40 +177,39 @@ def _torque_limit(self):
 
     def torque(self, currents):
         # Docstring of superclass
         mp = self._motor_parameter
-        return 1.5 * mp['p'] * (mp['l_m'] * currents[self.I_E_IDX] + (mp['l_d'] - mp['l_q']) * currents[self.I_SD_IDX]) * currents[self.I_SQ_IDX]
+        return 1.5 * mp['p'] * (mp['l_M'] * currents[self.I_E_IDX] * mp['i_k_rs'] + (mp['l_d'] - mp['l_q']) * currents[self.I_SD_IDX]) * currents[self.I_SQ_IDX]
 
     def electrical_jacobian(self, state, u_in, omega, *args):
         mp = self._motor_parameter
-        sigma = 1 - mp['l_m'] ** 2 / (mp['l_d'] * mp['l_e'])
         return (
             np.array([ # dx'/dx
-                [                                 -mp['r_s'] / mp['l_d'],                          mp['l_q'] / (sigma * mp['l_d']) * omega * mp['p'], mp['l_m'] * mp['r_e'] / (sigma * mp['l_d'] * mp['l_e']), 0],
-                [               -mp['l_d'] / mp['l_q'] * omega * mp['p'],                                                     -mp['r_s'] / mp['l_q'],                -omega * mp['p'] * mp['l_e'] / mp['l_q'], 0],
-                [mp['l_m'] * mp['r_s'] / (sigma * mp['l_d'] * mp['l_e']), -omega * mp['p'] * mp['l_m'] * mp['l_q'] / (sigma * mp['l_d'] * mp['l_e']),                                  -mp['r_e'] / mp['l_e'], 0],
-                [                                                      0,                                                                          0,                                                       0, 0],
+                [                       -mp['r_s'] / (mp['l_d'] * mp['sigma']),                          mp['l_q'] / (mp['sigma'] * mp['l_d']) * omega * mp['p'], mp['l_M'] * mp['r_e'] / (mp['sigma'] * mp['l_d'] * mp['l_E']) * mp['i_k_rs'], 0],
+                [                     -mp['l_d'] / mp['l_q'] * omega * mp['p'],                                                           -mp['r_s'] / mp['l_q'],                      -omega * mp['p'] * mp['l_E'] / mp['l_q'] * mp['i_k_rs'], 0],
+                [mp['l_M'] * mp['r_s'] / (mp['sigma'] * mp['l_d'] * mp['l_E']), -omega * mp['p'] * mp['l_M'] * mp['l_q'] / (mp['sigma'] * mp['l_d'] * mp['l_E']),                                        -mp['r_E'] / mp['l_E'] * mp['i_k_rs'], 0],
+                [                                                            0,                                                                                0,                                                                            0, 0],
             ]),
             np.array([ # dx'/dw
                 mp['p'] * mp['l_q'] / mp['l_d'] * state[self.I_SQ_IDX],
-                -mp['p'] * mp['l_d'] / mp['l_q'] * state[self.I_SD_IDX] - mp['p'] * mp['l_m'] / mp['l_q'] * state[self.I_E_IDX],
-                -mp['p'],
+                -mp['p'] * mp['l_d'] / mp['l_q'] * state[self.I_SD_IDX] - mp['p'] * mp['l_M'] / mp['l_q'] * state[self.I_E_IDX] * mp['i_k_rs'],
+                -mp['p'] * mp['l_M'] * mp['l_q'] / (mp['sigma'] * mp['l_d'] * mp['l_E']),
```

Comment:
-mp['p'] * mp['l_M'] * mp['l_q'] / (mp['sigma'] * mp['l_d'] * mp['l_E']) * state[self.I_SQ_IDX],

Reply:
Isn't there a / mp['sigma'] missing in line 204?